UnleashSampleApp/ViewController.swift

```diff
@@ -10,34 +10,58 @@ import Unleash
 
 // This constant is to emulate a client knowing its target environment is (e.g. QA, Production)
 struct Constants {
-    static let environment: String = "QA"
+  static let environment: String = "QA"
 }
 
 class ViewController: UIViewController {
-    //MARK: Properties
-    @IBOutlet internal weak var isFeatureEnabled: UILabel!
+  //MARK: Properties
+  @IBOutlet internal weak var isFeatureEnabled: UILabel!
+  @IBOutlet internal weak var activityIndicator: UIActivityIndicatorView!
+  private var isLoading: Bool = false {
+    didSet {
+      isLoading
+      ? activityIndicator.startAnimating()
+      : activityIndicator.stopAnimating()
+    }
+  }
+  private var unleash: Unleash!
+  
+  // MARK: - View Lifecycle
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(animated)
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        let unleash: Unleash = Unleash(appName: "ios-unleash-sample-app",
-                                       url: "https://unleash.silvercar.com/api",
-                                       refreshInterval: 300000,
-                                       strategies: [EnvironmentStrategy()])
-        
-        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
-            self.isFeatureEnabled?.text = "\(unleash.isEnabled(name: "enable-auth0-in-admin-client"))"
-        }
+    isLoading = true
+    unleash = Unleash(
+      appName: "ios-unleash-sample-app",
+      url: "https://unleash.silvercar.com/api",
+      refreshInterval: 300000,
+      strategies: [EnvironmentStrategy()])
+    unleash.delegate = self
+  }
+}
+
+// MARK: - UnleashDelegate
+extension ViewController: UnleashDelegate {
+  func unleashDidFetchToggles(_ unleash: Unleash) {
+    DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
```

Comment:
This is PURELY for show. Since this is a demo app, I wanted to bring attention to the spinner momentarily before the results appeared if the feature was enabled or not. Not a huge deal to have it in though haha.